internal/scan/scanner.go

```diff
@@ -118,6 +120,11 @@ func (dScanner *DebrickedScanner) Scan(o IOptions) error {
 		return nil
 	}
 
+	if dOptions.JsonFilePath != "" {
```

Comment:
Please add a test to `scanner_test.go` for these changes :blush:

Reply:
I´ve added a test case but was unable to verify it works since I´m on windows, could you have a look please?